packages/ui/Resizable/useResize.js

```diff
@@ -0,0 +1,63 @@
+import {forward} from '@enact/core/handle';
+import {useCallback, useContext, useEffect, useRef} from 'react';
+
+import ResizeContext from './ResizeContext';
+
+/**
+ * Object returned by `useResize`
+ * Object has a property named with the value of the resize property of config.
+ * And this property has resizehandler.
+ *
+ * @typedef {Object} useResizeInterface
+ * @memberof ui/Resizable
```

Comment:
Does below make sense?
```suggestion
 * @property {Function} resize The handler for resize
 * @memberof ui/Resizable
```